flash/core/data/io/input.py

```diff
@@ -171,8 +171,8 @@ def __hash__(self) -> int:
         return hash(self.value)
 
 
-class DefaultDataKeys(LightningEnum):
-    """The ``DefaultDataKeys`` enum contains the keys that are used by built-in data sources to refer to inputs and
+class InputDataKeys(LightningEnum):
```

Comment:
This also includes PREDS which is only in input rather than output. Maybe it should be renamed to just DATA_KEYS? or IO_KEYS maybe?